package.json

```diff
@@ -5,8 +5,9 @@
   "main": "main",
   "scripts": {
     "build:cjs": "tsc",
-    "build": "npm-run-all -p build:** -s dojo-package",
-    "clean": "rimraf dist coverage",
+    "build": "npm-run-all -p build:** -s generate:core dojo-package",
+    "clean": "rimraf dist coverage src/core",
+    "generate:core": "node ./src/v4/scripts/generate-dependency-map.js",
```

Comment:
This could be run with ts-node, allowing the files to be written in TS without needing to worry about a build step.